### DISCUSSION TITLE
Is ILI9486 supported

### DISCUSSION THREAD

**Discussion by spREGR** [HASwitchPlate/openHASP#63] [Hardware]
Hello, 

In the documentation, I can see ILI9341 supported. I would like to try the 3.5" version which has the model # ILI9486. Is this supported. 
Thank you

**Comment by fvanroie** [comment]
There is support for ili9486 in the [TFT_eSPI library](https://github.com/Bodmer/TFT_eSPI) that we use. But you need to provide a custom configuration for it like [this](https://github.com/fvanroie/hasp-lvgl/blob/4d7350d3996e78654814ab4c6ae2bf2d8ca609a3/user_setups/esp32/d132-unoshield_ili9486_parallel.ini).

What board and lcd are you using or planning to use?

**Reply by spREGR** [reply]
Hi
Thank you for the fast reply. I am planing to use ESP32 (Wemos)
The LCD I am looking at is [this](https://www.electronicscomp.com/3.5-inch-tft-lcd-touch-screen-display-shield-for-arduino-uno?gclid=Cj0KCQiAj9iBBhCJARIsAE9qRtBo_epFdIzxbwKOO_YZaJ9KbstbHOImffD5IZQKVgJZcFEMsX63EIwaAle2EALw_wcB)

Thank you

**Reply by fvanroie** [reply]
The display will work, but contrary to the specification in the link, [the LCD](http://www.lcdwiki.com/3.5inch_Arduino_Display-UNO) does *not* have a XPT2046 chip.
You will rather have to hack the board with jumper wires and use a custom touch library. See [this info](https://github.com/Bodmer/TFT_eSPI#8-bit-parallel-support)

If you need touch I would not recommend this combination.
